WordPress-iOS-Shared/Core/WPFontManager.m

```diff
@@ -172,7 +172,7 @@ + (UIFont *)merriweatherBoldFontOfSize:(CGFloat)size
 + (void)dynamicallyLoadFontResourceNamed:(NSString *)name
 {
     NSString *resourceName = [NSString stringWithFormat:@"%@/%@", kBundle, name];
-    NSURL *url = [[NSBundle mainBundle] URLForResource:resourceName withExtension:@"ttf"];
+    NSURL *url = [[NSBundle bundleForClass:[WPFontManager class]] URLForResource:resourceName withExtension:@"ttf"];
```

Comment:
Since you're in a static method in WPFontManager you might as well replace `[WPFontManager class]` with `self`.  This is not really a mistake though.